dockers/docker-orchagent/start.sh

```diff
@@ -27,9 +31,12 @@ ORCHAGENT_ARGS=""
 
 PORTSYNCD_ARGS=""
 
+SWSSCONFIG_ARGS="00-copp.config.json "
+
 if [ "$onie_platform" == "x86_64-dell_s6000_s1220-r0" ]; then
     ORCHAGENT_ARGS+="-m $MAC_ADDRESS"
     PORTSYNCD_ARGS+="-p /etc/ssw/Force10-S6000/port_config.ini"
+    SWSSCONFIG_ARGS+="td2.32ports.qos.json td2.32ports.buffers.json"
 elif [ "$aboot_platform" == "x86_64-arista_7050_qx32" ]; then
```

Comment:
arista platform should have the same SWSSCONFIG_ARGS since it is the same td2 platform.